t/unit/contrib/proj/foo.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import absolute_import, unicode_literals
 
 from celery import Celery, shared_task
-from xyzzy import plugh  # noqa
+from .xyzzy import plugh  # noqa
```

Comment:
Are we sure this is issue is not caused by something specific to the environment? Wouldn't we observe issues in Travis CI as well?